docs/08-developer-guide/04-project/02-release-guide.md

```diff
@@ -155,17 +189,18 @@ Once the previous two pull requests to update protolock files and the Ozone SNAP
 :::important
 The parent commit of the release branch should be the commit that was merged in the [proto.lock file update](#build-and-commit-protolock-files-to-the-master-branch).
 :::
-Name the branch after the major and minor version of the release, so patch releases can also be done off this branch. For example, If releasing 1.2.0, create a branch called `ozone-1.2` . All release related changes will go to this branch until the release is complete.
+Name the branch after the major and minor version of the release, so patch releases can also be done off this branch. For example, If releasing 2.0.0, create a branch called `ozone-2.0` . All release related changes will go to this branch until the release is complete.
 
 ### Set Up Local Environment
 
 The following variables will be referenced in commands:
 
 ```bash
-export VERSION=1.1.0 # Set to the version of ozone being released.
+export VERSION=2.0.0 # Set to the version of ozone being released.
 export RELEASE_DIR=~/ozone-release/ # ozone-release needs to be created
 export CODESIGNINGKEY=<your_gpg_key_id>
 export RC=0 # Set to the number of the current release candidate, starting at 0.
+export MAVEN_GPG_PASSPHRASE=<PASSPHRASE> # Maven passes this environment variable to gpg for passphrase.
```

Comment:
With `pinentry` installed, I think this is unnecessary.

Reply:
Without this parameter, gpg will ask for passphrase during the build, and it's annoying because it asks for several times during the build and you have to be in front of the console monitoring.